wgpu-core/src/instance.rs

```diff
@@ -134,9 +134,21 @@ impl<B: hal::Backend> Adapter<B> {
             adapter_features.contains(hal::Features::SAMPLER_ANISOTROPY),
         );
         extensions.set(
-            wgt::Extensions::TEXTURE_BINDING_ARRAY,
+            wgt::Extensions::SAMPLED_TEXTURE_BINDING_ARRAY,
```

Comment:
so this no longer applies to storage textures?

Reply:
I believe the gfx-rs feature _should_ allow us to use storage textures too, but in terms of wgpu-rs, we only ever implemented it for sampled textures. This can, ofc, be expanded.

Reply:
ok, that sounds fine by me